### ISSUE TITLE
update DDAH2 branch of PTHR12737 = DIMETHYLARGININE DIMETHYLAMINOHYDROLASE

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-annotation#4683]
I am creating this ticket on behalf of Dr. Vinitha Ragavan who submitted a request for correction of Ddah2 annotations via MGI's help desk. Her email is included below. 

* **PTHR ID & PTN node:**

PTHR12737 = DIMETHYLARGININE DIMETHYLAMINOHYDROLASE 
node = SF16-N(G),N(G)-DIMETHYLARGININE DIMETHYLAMINOHYDROLASE 2 (AN9) 

* **Sequences with problematic annotation (ID + gene/protein name):**

Here are the sequences with annotations to the MF term "**dimethylargininase activity**"
or to BP terms under "**arginine metabolism**". There may be additional BP problem annotations that don't map up to "**arginine metabolism**"

I have already deleted the single annotation MGI annotation which was based on the sequence alighment with DDAH1 and its enzymatic function of "**dimethylargininase activity**". I have also deleted the PINC annotations as these often do not meet current standards and some were redundant with UniProt annotations from the same paper. 

Taxon   name | Feature ID | Symbol | Gene name | Aspect | GO ID | GO term | Contributor | Ev Code | With | Reference | Date
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Mus musculus | MGI:MGI:1859016 | Ddah2 | dimethylarginine dimethylaminohydrolase 2 | P | GO:0006527 | arginine catabolic process | MGI | ISA | EMBL:AF004106 | PMID:10493931 | 20000616
Rattus norvegicus | RGD:1302955 | Ddah2 | dimethylarginine dimethylaminohydrolase 2 | F | GO:0016403 | dimethylargininase activity | RGD | IDA |   | PMID:17322279\|RGD:1625578 | 20070614
Rattus norvegicus | RGD:1302955 | Ddah2 | dimethylarginine dimethylaminohydrolase 2 | P | GO:0006525 | arginine metabolic process | RGD | IDA |   | PMID:17322279\|RGD:1625578 | 20070614
Rattus norvegicus | RGD:1302955 | Ddah2 | dimethylarginine dimethylaminohydrolase 2 | F | GO:0016403 | dimethylargininase activity | UniProt | ISS | UniProtKB:O95865 | GO_REF:0000024\|RGD:1600115 | 20100804
Homo sapiens | UniProtKB:O95865 | DDAH2 | N(G),N(G)-dimethylarginine dimethylaminohydrolase 2 | F | GO:0016403 | dimethylargininase activity | UniProt | IDA |   | PMID:10493931 | 20100804
Homo sapiens | UniProtKB:O95865 | DDAH2 | N(G),N(G)-dimethylarginine dimethylaminohydrolase 2 | F | GO:0016403 | dimethylargininase activity | Reactome | TAS |   | Reactome:R-HSA-5693373 | 20220504
Homo sapiens | UniProtKB:O95865 | DDAH2 | N(G),N(G)-dimethylarginine dimethylaminohydrolase 2 | P | GO:0006527 | arginine catabolic process | PINC | TAS |   | PMID:10493931 | 20030904
Bos taurus | UniProtKB:Q3SX44 | DDAH2 | N(G),N(G)-dimethylarginine dimethylaminohydrolase 2 | F | GO:0016403 | dimethylargininase activity | UniProt | ISS | UniProtKB:O95865 | GO_REF:0000024 | 20100804

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

New experimental work clarifying a situation which has had conflicting experimental results:

> Date: Tuesday, July 11, 2023 at 6:30 AM
> Vinitha Ragavan's Comments:
>  
> Dear Mouse Genome Informatics Help Team,
> 
> With this email we are informing you about a major discrepancy between the information in your database and the current state of knowledge regarding the function of dimethylarginine dimethylaminohydrolase 2 (DDAH2), Access # - MGI:1859016 (https://www.informatics.jax.org/go/marker/MGI:1859016?header=hydrolase)
> It has just been shown in the multicenter consortium study that DDAH2 has nothing to do with metabolism of the endogenous dimethylarginines (https://pubmed.ncbi.nlm.nih.gov/37296100/). The international consortium involved 14 different universities worldwide and the consortium paper was published in Nature Communications.
> As a result, the current substrate of DDAH2 is unknown. We kindly ask you to correct the database to avoid unnecessary confusion.
> 
> Sincerely,
> Vinitha Ragavan

* **New annotations to incorporate**

I have curated the paper Dr. Ragavan brought to our attention, PMID:37296100, and generated NOT annotations for both the human DDAH2 (in Protein2GO) and mouse Ddah2 genes (in Noctua). Since I have curated the human and mouse annotations in different curation interfaces, they will likely become available for use in PAINT at different times.

![20230714-Ddah2-NOTannotations](https://github.com/geneontology/go-annotation/assets/10533218/d3f79d71-d32a-45ff-963d-1f51d71490bf)

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by marcfeuermann** [comment]
Thanks Karen for sharing this. I've blocked all corresponding annotations at the DDAH2 branch. This is a nice example of ancient duplication followed by loss of function. We will close this ticket once all experimental annotations will be corrected.
Best regards,
Marc.

**Comment by krchristie** [comment]
Thanks for getting to this so quickly Marc,

Best regards, 

Karen
